client/src/Components/Navbar.jsx

```diff
@@ -0,0 +1,162 @@
+import React, { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link, useNavigate, useParams } from 'react-router-dom'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {faGlobe} from '@fortawesome/free-solid-svg-icons'
+import i18 from 'i18next'
+import { useAuth0 } from "@auth0/auth0-react";
+
+
+const Navbar = () => {
+  const [dropdownVisible, setDropdownVisible] = useState(false);
+  const {t} =  useTranslation();
+  const {lng} = useParams(); //capture the language from url
+  const navigate = useNavigate();
+  const { user, isLoading, isAuthenticated, loginWithRedirect, logout } = useAuth0();
+
+  const {ul1,ul2,ul3,ul4,ul5} = t('ulTranslation')
+  const {login,register,logOut} = t('logRegTranslation')
+  const {Technology,Music,Literature,Politics,Culture} = t('blogTopics')
+
+  useEffect(()=>{
+    const savedLang = localStorage.getItem('i18nextLng') || 'en';
+    i18.changeLanguage(savedLang);
+  },[])
+  
+  const handleLanguageChange = (lang) => {
+    const currentPath = window.location.pathname;
+    const newPath = currentPath.replace(`/${i18.language}`, `/${lang}`);
+    i18.changeLanguage(lang);
+    localStorage.setItem('i18nextLng', lang);
+    navigate(newPath);
+    setDropdownVisible(false);
+  };
+  
+  // const getLanguagePath = () => `/${i18.language}`
+
+  if(isLoading)
+  {
+    return <div>Loading...</div>
+  }
+
+  return (
+    <div className='gap-[1150px]'>
+
+      {/* This is the 1st navbar */}
+      <div className= "bg-green-700}">
```

Comment:
This is hardly readable, why can't `className` be something meaningful like the section it represents and the styling part `bg-green-700` separate from the name?

Reply:
For the developer working on `client-side`, I think it is highly readable, because the `tailwind` is much `efficient` and `readable` way of adding `styling` onto the frontside.